x-pack/plugins/siem/public/pages/overview/overview.tsx

```diff
@@ -7,23 +7,43 @@
 import { EuiFlexGroup } from '@elastic/eui';
 import React from 'react';
 import { pure } from 'recompose';
+import chrome from 'ui/chrome';
 
 import { OverviewHost } from '../../components/page/overview/overview_host';
 import { OverviewNetwork } from '../../components/page/overview/overview_network';
 import { GlobalTime } from '../../containers/global_time';
 
 import { Summary } from './summary';
+import { EmptyPage } from '../../components/empty_page';
+import { WithSource, indicesExistOrDataTemporarilyUnavailable } from '../../containers/source';
+import { IndexType } from '../../graphql/types';
+
+import * as i18n from './translations';
+
+const basePath = chrome.getBasePath();
 
 export const OverviewComponent = pure(() => (
-  <>
-    <GlobalTime>
-      {({ to, from, setQuery }) => (
-        <EuiFlexGroup>
-          <Summary />
-          <OverviewHost endDate={to} startDate={from} setQuery={setQuery} />
-          <OverviewNetwork endDate={to} startDate={from} setQuery={setQuery} />
-        </EuiFlexGroup>
-      )}
-    </GlobalTime>
-  </>
+  <WithSource sourceId="default" indexTypes={[IndexType.FILEBEAT, IndexType.PACKETBEAT]}>
```

Comment:
I think we want `IndexType.AUDITBEAT` here as well.

https://github.com/elastic/kibana/blob/feature-secops/x-pack/plugins/siem/server/lib/overview/query.dsl.ts#L101